students/Zach Cooper/lesson03/basic_operations.py

```diff
@@ -0,0 +1,103 @@
+""" Basic Operations"""
+import logging
+import peewee as pw
+from customers_model import Customer
+
+
+logging.basicConfig(level=logging.INFO)
+LOGGER = logging.getLogger(__name__)
+
+
+def add_customer(customer_id, first_name, last_name, home_address,
+                 phone_number, email_address, status, credit_limit):
+    """
+        Returns new customer with **kwargs info
+    """
+    try:
+        credit_limit = float(credit_limit)
+    except ValueError as err:
+        logging.error(err)
+        raise
+    new_customer = Customer.create(
+        customer_id=customer_id,
+        first_name=first_name,
+        last_name=last_name,
+        home_address=home_address,
+        phone_number=phone_number,
+        email_address=email_address,
+        status=status,
+        credit_limit=credit_limit)
+    new_customer.save()
+
+    LOGGER.info('Customer database add successful')
+    LOGGER.info('%s %s has been created', Customer.first_name,
+                Customer.last_name)
+
+
+def search_customer(customer_id):
+    """
+        Search for customer name in database
+        :return: Customer info if Ture, empty dict if None
+        :param customer_id: customer_id to search for
+    """
+    find_customer_dict= {}
+    LOGGER.info("Search for customer name in database...")
+    try:
+        find_customer = Customer.get(Customer.customer_id == customer_id)
+        if find_customer:
+
+            find_customer_dict = {"first_name": find_customer.first_name,
+                                  "last_name": find_customer.last_name,
+                                  "email_address": find_customer.email_address,
+                                  "phone_number": find_customer.phone_number}
+
+        LOGGER.info('%s %s is in the database. There phone # is %s',
+                    find_customer.first_name, find_customer.last_name,
+                    find_customer.phone_number)
+
+    except pw.DoesNotExist:
+        LOGGER.info('No customer with customer id of %s', Customer.customer_id)
+
+        return find_customer_dict
+
+
+def delete_customer(customer_id):
+    """
+        Deletes customer from database
+        :return: empty dict if custome doesnt exist
+        :param customer_id: customer_id to search and delete for
+    """
+    LOGGER.info("Creating new customer named Tom and then deleting him")
+    try:
+        customer_remove = Customer.get(Customer.customer_id == customer_id)
+        customer_remove.delete_instance()
+
+    except pw.DoesNotExist:
+        return None
+    LOGGER.info("Customer successfully deleted in database")
```

Comment:
Also true in the rest of these. The only thing that will give a does not exist is the get.